### PR TITLE
Dashboard labels

### PR DESCRIPTION
**Pull request by dnil**
This PR fixes a small layout issue.

From 
Screenshot 2022-11-22 at 10 26 21
To 
Screenshot 2022-11-22 at 10 26 44

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
-## [unreleased]
+## [4.61]
```
Comment (northwestwitch) [inline CHANGELOG.md:7]: Did you start this PR from the new minor release branch?

**Review by northwestwitch — CHANGES_REQUESTED** [review]
Looks good, but I think you started from the wrong branch!

File: CHANGELOG.md (hunk 1 above)
Comment (dnil) [reply CHANGELOG.md:7]: Ooh, quite possible! Will check!

**Comment by dnil** [comment]
Looks like that worked, but maybe everything is just working again? Anyway, branch should be rebased now!

File: requirements-dev.txt [hunk 2]
```diff
@@ -17,3 +17,6 @@ black
 mkdocs
 mkdocs-material
 markdown-include
+
+# Setup
```
Comment (northwestwitch) [inline requirements-dev.txt:21]: I'm not sure if it's the fix. Let's see in another PR

**Review by northwestwitch — APPROVED** [review]
I approve because the PR fixes the labels!

File: requirements-dev.txt (hunk 2 above)
Comment (dnil) [reply requirements-dev.txt:21]: Ok, I'll wait with merging. No use having it there if it was just luck.

Comment (dnil) [reply requirements-dev.txt:21]: Looks rather promising!

**Pull request merged**